front-end/src/api/func.js

```diff
@@ -4,14 +4,14 @@ const { post, get } = request
 
 export const funcApi = {
   list: '/admin/v3/functions/public/default',
-  create: '/function/create',
+  create: '/admin/v3/functions/public/default/myfunc',
   info: ({ funcName }) => `/admin/v3/functions/public/default/${funcName}`,
   stats: ({ funcName }) => `/admin/v3/functions/public/default/${funcName}/stats`,
   status: ({ funcName }) => `/admin/v3/functions/public/default/${funcName}/status`,
   trigger: ({ funcName }) => `/admin/v3/functions/public/default/${funcName}/trigger`,
-  deleteFunc: ({ funcName }) => `/admin/v3/functions/public/default/${funcName}/delete`,
```

Comment:
This URL is incorrect. It does not need to contain `delete` in the delete URL. Please check with the postman.